example/provider-local/seed-base/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Can we rename this folder from `example/provider-local/seed-base` to `example/provider-local/seed-kind`? Then we can rename `seed2` to `seed-kind2` (see below)